databasebuild.yaml

```diff
@@ -0,0 +1,19 @@
+# using this will require buildcomponents.yaml 
```

Comment:
This sounds a bit ambiguous, like if you include this, it will automatically include build components (which of course is the opposite of what you mean).